include/fmt/core.h

```diff
@@ -1146,6 +1146,9 @@ constexpr bool is_arithmetic_type(type t) {
 struct unformattable {};
 struct unformattable_char : unformattable {};
 struct unformattable_const : unformattable {};
+struct unformattable_function_pointer : unformattable {};
+struct unformattable_member_function_pointer : unformattable {};
+struct unformattable_member_object_pointer : unformattable {};
```

Comment:
Those are just different kinds of pointers so let's just use `unformattable_pointer` for simplicity.